QA.md

```diff
@@ -0,0 +1,210 @@
+| Robustness                       |             |                 |              |              |     |             |                 |                |
+| - Safe under perturbations       |             |                 |              |              |     |             |                 |                |
+| - Performant after perturbations |             |                 |              |              |     |             |                 |                |
+
+### Application integration (ABCI)
+
+| Concern                          | Code Review | Static Analysis | Unit Testing | Fuzz Testing | MBT | E2E Testing | Testnet Testing | Protocol Audit |
+|----------------------------------|-------------|-----------------|--------------|--------------|-----|-------------|-----------------|----------------|
+| Code/specification alignment     |             |                 |              |              |     |             |                 |                |
+| - Grammar                        |             |                 |              |              |     |             |                 |                |
+| - Parameters of API functions    |             |                 |              |              |     |             |                 |                |
+| Integration with other protocols |             |                 |              |              |     |             |                 |                |
+| Performance                      |             |                 |              |              |     |             |                 |                |
+| Robustness                       |             |                 |              |              |     |             |                 |                |
+| - Safe under perturbations       |             |                 |              |              |     |             |                 |                |
+| - Performant after perturbations |             |                 |              |              |     |             |                 |                |
+| Correct usage upholds liveness   |             |                 |              |              |     |             |                 |                |
+
+### Network stability
+
+### Operational dynamics
+
+#### Crash/recovery
+
+#### Upgrading
+
+## QA Log
+
+In this section, we aim to capture all of our one-off QA efforts. Links to the
+results of the following types of efforts can and should be added under the
+[Log](#log) section in chronological order, with the _most recent entries
+first_:
+
+- Audit results
+- One-off testnet executions
+- Model checker runs
+- Experiment results
+- Any other manual QA effort you feel is important to share with the community
+
+When capturing QA efforts, please use the following format:
+
+```markdown
+- YYYY-mm-dd: Descriptive title
+  - A description of the QA effort.
+  - References:
+    - [Link 1]
+    - [Link 2]
```

Comment:
Please add a field to this template with the outcome: "PASS, FAIL, INCONCLUSIVE".
The "INCONCLUSIVE" field is typically used when you try to run a test case (e.g. using a complex testnet script), and the test cannot be run because of problems outside the scope of the UUT.

Reply:
I like the "outcome" idea. I am not sure whether we should give a list to choose from. For instance, if the item describes an audit, then we could say "We opened the following issues"